jaxopt/_src/lbfgs.py

```diff
@@ -255,7 +264,9 @@ def init_state(self,
                       s_history=init_history(init_params, self.history_size),
                       y_history=init_history(init_params, self.history_size),
                       rho_history=jnp.zeros(self.history_size, dtype=dtype),
-                      aux=aux)
+                      gamma=jnp.asarray(1.0, dtype=dtype),
+                      aux=aux,
+                      failed_linesearch=jnp.asarray(False))
```

Comment:
Without `asarray` I guess?

Reply:
yup changing

Reply:
Thanks for checking. I see that `bool(jnp.asarray(True))` gives `True` and similarly for `False` so all is good.